notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1916 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    RNG = random.PRNGKey(0)
</pre><p>Maybe it is better to use different RNG keys for getting priors, getting posterior samples?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>